plugwise/util.py

```
@@ -28,6 +29,7 @@
     SpecialType,
     SwitchType,
 )
+from plugwise.exceptions import DataMissingError
```

Ruff (0.8.2)

`plugwise.exceptions.DataMissingError` imported but unused

Remove unused import: `plugwise.exceptions.DataMissingError`

(F401)